edx_proctoring/tests/test_views.py

```diff
@@ -2680,13 +2680,32 @@ def test_can_delete_user(self):
         response = self.client.delete(deletion_url)
         assert response.status_code == 200
         data = response.json()
+        # if there is no user data, then no deletion happens
+        assert data == {}
+
+        proctored_exam = ProctoredExam.objects.create(
+            course_id='a/b/c',
+            content_id='test_content',
+            exam_name='Test Exam',
+            external_id='123aXqe3',
+            is_proctored=True,
+            is_active=True,
+            time_limit_mins=90,
+            backend='test',
+        )
+        create_exam_attempt(proctored_exam.id, self.second_user.id, True)
+
+        response = self.client.delete(deletion_url)
+        assert response.status_code == 200
+        data = response.json()
+        # if there is an attempt, we'll try to delete from the backend
         assert data == {'test': True}
         test_backend = get_backend_provider(name='test')
         assert test_backend.last_retire_user is not None
 
         # running a second time will return a false status
```

Comment:
Any idea if we will have a way to separate out the "no user to delete" case from a real error? We frequently have to re-run users and the way it's written currently will add a fair amount of work for us.

Reply:
"no user to delete" will be a 200 response with `"{}"` as content

"deleted user " will be a 200 response with `"{"backend": true}"`